apps/purchase/models.py

```diff
@@ -89,8 +89,8 @@ class PurchaseInvoiceOrders(OrderNumberMixin):
     vendor_agent_id = models.ForeignKey(VendorAgent, on_delete=models.CASCADE, null=True, default=None, db_column = 'vendor_agent_id')
     
```

Comment:
Can you add below 2 columns  to DB, Model and code as well,

shipping_address VARCHAR(1024),
billing_address VARCHAR(1024),

These 2 fields should be part of payload(CREATE and PUT)

Reply:
Okay Thulasi.